canonical-websites/insights.ubuntu.com
# Add prometheus metrics, to help with debugging

**Based on #225, review that first**

QA
--

``` bash
./run build  # Install depndencies
./run exec --expose-port 8023 --expose-port 9990 --env FLASK_APP=app.py flask run --host 0.0.0.0 --port 8023  # Run server & prometheus server
```

Now go to http://0.0.0.0:8023/, browse around a bit.

Now go to http://0.0.0.0:9990/, see prometheus metrics for both normal requests (`flask_http_request_duration_*`) and feed requests (`feed_request_latency_seconds_*`).
